src/Microsoft.TemplateEngine.Abstractions/PublicAPI.Unshipped.txt

```diff
@@ -1,5 +1,9 @@
-﻿Microsoft.TemplateEngine.Abstractions.ITemplate.TemplateSourceRoot.get -> Microsoft.TemplateEngine.Abstractions.Mount.IDirectory!
+Microsoft.TemplateEngine.Abstractions.ITemplate.TemplateSourceRoot.get -> Microsoft.TemplateEngine.Abstractions.Mount.IDirectory!
 Microsoft.TemplateEngine.Abstractions.IVariableCollection
 Microsoft.TemplateEngine.Abstractions.IVariableCollection.Parent.get -> Microsoft.TemplateEngine.Abstractions.IVariableCollection?
 Microsoft.TemplateEngine.Abstractions.IVariableCollection.Parent.set -> void
-Microsoft.TemplateEngine.Abstractions.ITemplateInfo.PreferDefaultName.get -> bool
\ No newline at end of file
+Microsoft.TemplateEngine.Abstractions.ITemplateInfo.PreferDefaultName.get -> bool
+Microsoft.TemplateEngine.Abstractions.Parameters.ParameterSetDataExtensions
+static Microsoft.TemplateEngine.Abstractions.Parameters.ParameterSetData.Empty.get -> Microsoft.TemplateEngine.Abstractions.Parameters.IParameterSetData!
+static Microsoft.TemplateEngine.Abstractions.Parameters.ParameterSetDataExtensions.GetValue(this Microsoft.TemplateEngine.Abstractions.Parameters.IParameterSetData! data, string! parameterName) -> Microsoft.TemplateEngine.Abstractions.Parameters.ParameterData!
+static Microsoft.TemplateEngine.Abstractions.Parameters.ParameterSetDataExtensions.TryGetValue(this Microsoft.TemplateEngine.Abstractions.Parameters.IParameterSetData! data, string! parameterName, out Microsoft.TemplateEngine.Abstractions.Parameters.ParameterData? parameterData) -> bool
```

Comment:
are those changes needed? seems irrelevant to the change

Reply:
The one on about the `PreferDefaultName` was added automatically to the file by the IDE. So I'm assuming that is a standard for the code. The rest came from the main branch, so I'm not sure what they are about (and not sure why they are showing separately)